src/pages/contact_us/_offices.tsx

```diff
@@ -416,7 +436,7 @@ export const Offices = () => {
                         </Flex>
                     </GridLayout>
                     <Flex fd="column" mt="30px" id="malaysia">
-                        <ContinentHeader>{localize('ASIA')}</ContinentHeader>
+                        <ContinentHeader>{localize('Asia    ')}</ContinentHeader>
```

Comment:
Do we need extra spaces after `Asia     `?

Reply:
Not needed, I didn't notice that. Thanks